modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/view_entry_content_detail.jsp

```diff
@@ -248,7 +249,13 @@ BlogsPortletInstanceConfiguration blogsPortletInstanceConfiguration = BlogsPortl
 							<liferay-asset:asset-tags-summary
 								className="<%= BlogsEntry.class.getName() %>"
 								classPK="<%= entry.getEntryId() %>"
-								portletURL="<%= renderResponse.createRenderURL() %>"
+								portletURL="<%=
```

Comment:
@tototrinh 
Please double check if the redirect will be BLANK as default and if this function work properly with the BLANK redirect.